src/pages/steps/ContactInfo.tsx

```diff
@@ -90,43 +78,37 @@ const ContactInfo = ({
                 <Stack flex={1}>
                   <Text fontWeight="bold">{contactInfoTypeLabel}</Text>
                   <Stack>
-                    {mergedContactAndBookingInfo[key].length === 0 && (
+                    {mergedInfo[key].length === 0 && (
                       <Text variant={TextVariants.MUTED}>
                         {t('create.additionalInformation.contact_info.empty', {
                           contactInfoType: contactInfoTypeLabel.toLowerCase(),
                         })}
                       </Text>
                     )}
-                    {mergedContactAndBookingInfo[key].map(
-                      (contactInfo, index) => {
-                        return (
-                          <Stack
-                            key={index}
-                            css={
-                              index !== 0
-                                ? css`border-top: 1px solid ${borderColorFromTheme}}`
-                                : ''
-                            }
-                          >
-                            <Inline spacing={3} alignItems="center">
-                              <Text
-                                key={index}
-                                paddingTop={2}
-                                paddingBottom={2}
-                              >
-                                {contactInfo}
-                              </Text>
-                              {isUsedForReservation(contactInfo) && (
-                                <Icon
-                                  name={Icons.TICKET}
-                                  color={iconColorFromTheme}
-                                />
-                              )}
-                            </Inline>
-                          </Stack>
-                        );
-                      },
-                    )}
+                    {mergedInfo[key].map((contactInfo, index) => {
+                      return (
+                        <Stack
+                          key={index}
+                          css={
+                            index !== 0
+                              ? css`border-top: 1px solid ${borderColorFromTheme}}`
+                              : ''
```

Comment:
```suggestion
                            index !== 0
                              ? css`border-top: 1px solid ${borderColorFromTheme}}`
                              : css''
```

the css prop expects a type specific to styled components. `css``` translates the empty string to the correct type.